services/space.controller.js

```diff
@@ -95,7 +114,39 @@ class SpaceServices extends eventEmiter{
             const arr = response.slice(searchTerm.pagination.skip , searchTerm.pagination.skip + searchTerm.pagination.limit )
             res.status(200).set(`Content-Pages`, maxPages ).json(arr)
         }catch(err){
-            res.status(400).json(err)
+            res.status(400).json(err.message)
+        }
+    }
+
+    getSpaceRegisteredTenant = async (req, res) => {
+        const {state} = req.query
+        let response, inDate, finDate  
+        const searchTerm = searchTermConstructor(req.query)            
+        try{ 
+            switch(state){
+                case 'current':
+                    inDate = moment().format("YYYY-MM-DD")
+                    finDate = moment().add(1,'days').format("YYYY-MM-DD")
+                    
+                    break
+                case 'reserved':
+                    break
+                case 'incoming':
+                    break
+                default:                 
```

Comment:
it was leave like this because we are expecting to amplify the functionality in the near future